docs/README.md

```diff
@@ -37,5 +37,7 @@ When installed for development, via Composer, the package also requires:
 
 
 
-## License
-This repository is a free software, and is released under the terms of the GNU General Public License version 2 or (at your option) any later version. See [LICENSE](https://github.com/inpsyde/modularity/blob/master/LICENSE) for complete license.
+## Copyright and License
+
+This package is [free software](https://www.gnu.org/philosophy/free-sw.en.html) distributed under the terms of the GNU General Public License version 2 or (at your option) any later version. For the full license, see [LICENSE](./LICENSE).
```

Comment:
The `./LICENSE` path does not work on GitHub pages built from the `docs/` folder. I would recommend using an absolute path to the root file, but maybe use `HEAD` so we are free to change the name of the default branch if we wanted.

```suggestion
This package is [free software](https://www.gnu.org/philosophy/free-sw.en.html) distributed under the terms of the GNU General Public License version 2 or (at your option) any later version. For the full license, see [LICENSE](https://github.com/inpsyde/modularity/blob/HEAD/LICENSE).
```

Reply:
Changed 🙂